week-1/Homework/mandatory/4-tax.js

```diff
@@ -17,7 +19,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+
+function formatCurrency(formated) {
+   
+  return "£" + (formated + formated*0.2).toFixed(2);
+}
```

Comment:
Read the instruction Sulaiman. You don't need to reproduce the calculation of the tax. Remember of the nested function. You have one already making that for you.